app/src/androidTest/java/ch/hikemate/app/navigation/BottomBarNavigationTest.kt

```diff
@@ -0,0 +1,84 @@
+package ch.hikemate.app.navigation
+
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import ch.hikemate.app.ui.navigation.BottomBarNavigation
+import ch.hikemate.app.ui.navigation.LIST_TOP_LEVEL_DESTINATIONS
+import ch.hikemate.app.ui.navigation.Route
+import ch.hikemate.app.ui.navigation.TEST_TAG_BOTTOM_BAR
+import ch.hikemate.app.ui.navigation.TEST_TAG_MENU_ITEM_PREFIX
+import ch.hikemate.app.ui.navigation.TopLevelDestinations
+import org.junit.Assert.assertEquals
+import org.junit.Assert.fail
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class BottomBarNavigationTest {
+  // Set up the Compose test rule
+  @get:Rule val composeTestRule = createComposeRule()
+
+  @Test
+  fun clickingOnAItemChangeScreen() {
```

Comment:
Nitpick: I believe there's an `n` and a `s` missing here

```suggestion
  fun clickingOnAnItemChangesScreen() {
```

Reply:
Good catch!!